Src/DependencyCollector/Microsoft.AI.DependencyCollector.Tests/MockCorrelationIdLookupHelper.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Microsoft.ApplicationInsights.DependencyCollector
+{
+    using Microsoft.ApplicationInsights.Common;
+    using System.Collections.Generic;
+
+    internal class MockCorrelationIdLookupHelper : ICorrelationIdLookupHelper
+    {
+        private readonly Dictionary<string, string> instrumentationKeyToCorrelationIdMap = new Dictionary<string, string>();
+
+        public void SetCorrelationId(string instrumentationKey, string correlationId)
```

Comment:
Niggle - you could also just have a constructor that takes the full content of the dictionary in one go and sets things up. Most likely when you are setting up a test - you already know all possible values. It is arguably cleaner to avoid creation and population being separate steps.